components/Banner/tests/unit/BannerViewTests.m

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Please update the copyright date.

Reply:
Addressed in f48539c3d7cd07ad36955c1a13b11c89ebecaaa4.